source/programme-talks.rst

```diff
@@ -0,0 +1,74 @@
+Programme of talks
+==================
+
+At PyCon UK 2015, we had approximately 70 talks.
+
+About 57 of the speakers were male, and the vast majority were of course
+extremely white.
+
+Some of our best friends are white men, and there is nothing wrong with
+us^wthem, but all the same some of our best friends are not, and for some reason
+the ones who aren't didn't come forward to speak at PyCon.
```

Comment:
I know what you're saying here, but find "aren't didn't" extremely hard to parse.  Also, "us^wthem" suggests that the conference committee is all white me, which it's not.

For this kind of writing, I think it's important to be as clear as possible.

What's wrong with saying "We are concerned that the conference line-up has been too white and too male, and we plan to work hard to ensure this is not the case in 2016."?